stdlib/contrib/README.md

```diff
@@ -15,8 +15,8 @@ Please see the [CONTRIBUTING](https://github.com/influxdata/flux/blob/master/CON
 
 ## Package Ownership
 
-Packages in the `contrib` directory are owned and maintained by their author not the InfluxData team.
-As such the author will be requested for review on all changes to the package.
+Packages in the `contrib` directory are owned and maintained by their author.
+As such, the InfluxData team requests the author for review on all changes to the package.
```

Comment:
As such, the InfluxData team will request a review by the original author on all changes proposed to the package.